packages/server/graphql/types/Team.ts

```diff
@@ -40,6 +40,11 @@ const Team: GraphQLObjectType = new GraphQLObjectType<ITeam, GQLContext>({
       type: new GraphQLNonNull(GraphQLID),
       description: 'A shortid for the team'
     },
+    autoJoin: {
```

Comment:
We want to be moving these to the public types, but when I do so, I get: `Team.autoJoin defined in resolvers, but not in schema`, but we don't need to define it in a resolver yet. The codegen file imports the `Team` type from [here](https://github.com/ParabolInc/parabol/blob/40fe222a6108205768bae3918317a36b49319c01/packages/server/postgres/queries/getTeamsByIds.ts#L6) where `autoJoin` is defined 🤔

Reply:
If I remove `autoJoin` from here and then add it here: 

https://github.com/ParabolInc/parabol/blob/03c20567f59f3d942c640c3c4c893502d1b4932e/packages/server/graphql/public/typeDefs/Team.graphql#L4

It works for me

Can you try?

Reply:
I'm not sure what was happening before, but that is working now. Thank you, Sir!